### PR TITLE
Changes to be able to build as Java 11 project

### PR DESCRIPTION
**Pull request by obfischer**
Changes to the project to be able to run it as Java 11 application.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -51,6 +51,8 @@
         <jqa-asciidoctor-utilities.version>1.13.0-SNAPSHOT</jqa-asciidoctor-utilities.version>
         <jqa-own-constraints.version>1.13.0-SNAPSHOT</jqa-own-constraints.version>
         <slf4j.version>1.8.0-beta4</slf4j.version>
+        <xo.version>1.4.2</xo.version>
```
Comment (StephanPirnbaum) [inline pom.xml]: Specified in uber-parent, not needed here

File: pom.xml [hunk 2]
```diff
@@ -185,6 +187,12 @@
             </dependency>
             <!-- End of misc dependencies -->
 
+            <dependency>
```
Comment (StephanPirnbaum) [inline pom.xml:188]: Why was this added here

**Review by DirkMahler — APPROVED** [review]
